go/l1client/txhandler/handler.go

```diff
@@ -0,0 +1,168 @@
+package txhandler
```

Comment:
Conceptually, this package handles the interaction with the management contract.
Let's add this to the readme, and also let's rename the types.
Package name proposal: "ethmngmtctr" (ethereum management contract)

Reply:
That's a good point. Can I suggest `rollupcontractlib` ? 
I have this idea that we were going to use rollup contract instead of management contract. Is that wrong ?

Reply:
done 👍